build.psm1

```diff
@@ -347,7 +347,9 @@ function Start-PSBuild {
         Write-Warning @"
```

Comment:
@iSazonov, your last commit had 9 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Get-Content.Check Get-Content containing multi-byte chars.Should return last three lines two lines at a time for -ReadCount 2 and -Tail 3
```powershell
Expected $null or empty, but got @(@{InputObject=System.Object[]; SideIndicator==>}, @{InputObject=System.Object[]; SideIndicator==>}, @{InputObject=System.Object[]; SideIndicator=<=}, @{InputObject=System.Object[]; SideIndicator=<=}).
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1: line 341
341:             Compare-Object -ReferenceObject $expected -DifferenceObject $result | Should -BeNullOrEmpty
```
Get-Content.Check Get-Content containing multi-byte chars.Should return first three lines two lines at a time for -TotalCount 3 and -ReadCount 2
```powershell
Expected $null or empty, but got @(@{InputObject=System.Object[]; SideIndicator==>}, @{InputObject=System.Object[]; SideIndicator==>}, @{InputObject=System.Object[]; SideIndicator=<=}, @{InputObject=System.Object[]; SideIndicator=<=}).
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1: line 353
353:             Compare-Object -ReferenceObject $expected -DifferenceObject $result | Should -BeNullOrEmpty
```
Export-FormatData.Can export all types
```powershell
Expected $null, but got @(@{InputObject=Microsoft.PowerShell.Commands.X509StoreLocation; SideIndicator==>}, @{InputObject=System.Security.Cryptography.X509Certificates.X509Certificate2; SideIndicator==>}, @{InputObject=System.Security.Cryptography.X509Certificates.X509Store; SideIndicator==>}, @{InputObject=System.CodeDom.Compiler.CompilerError; SideIndicator==>}, @{InputObject=System.Reflection.Assembly; SideIndicator==>}, @{InputObject=System.Reflection.AssemblyName; SideIndicator==>}, @{InputObject=System.Globalization.CultureInfo; SideIndicator==>}, @{InputObject=System.Diagnostics.FileVersionInfo; SideIndicator==>}, @{InputObject=System.Diagnostics.EventLogEntry; SideIndicator==>}, @{InputObject=System.Diagnostics.EventLog; SideIndicator==>}).
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Export-FormatData.Tests.ps1: line 30
30:             Compare-Object $fd $actualAllFormat | Should -Be $null
```
Select-String.String actions.Should use the ca alias for casesensitive
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1: line 48
48:             $equal | Should -BeTrue
```
Stream writer tests.Write-Information cmdlet.Write-Information works with defaults
```powershell
Expected 0, but got 2.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1: line 115
115:             (Compare-Object $result $returnValue -SyncWindow 0).length | Should -Be 0
```